<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addComp</class>
 <widget class="QDialog" name="addComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Computer_made">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Made</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="click_computer_yes">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Computer_Type">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Please enter computer type...</string>
   </property>
  </widget>
  <widget class="QLabel" name="Computer_Buildyear">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Build Year</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Computer">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>221</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Computer_Build_Year">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Build Year..</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Input_Computer_Name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Please Enter Name ....</string>
   </property>
  </widget>
  <widget class="QLabel" name="Computer_name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="click_computer_no">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="Computer_Type">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Name_empty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Type_empty">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Made_Not_set">
   <property name="geometry">
    <rect>
     <x>74</x>
     <y>120</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Build_Year_empty">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>160</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Build_Year_int">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>Input_Computer_Name</tabstop>
  <tabstop>Input_Computer_Type</tabstop>
  <tabstop>click_computer_yes</tabstop>
  <tabstop>click_computer_no</tabstop>
  <tabstop>Input_Computer_Build_Year</tabstop>
  <tabstop>Add_Computer</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
